elasticsearch/src/main/java/org/hibernate/search/elasticsearch/util/impl/AutoGrowingPagedBuffer.java

```diff
@@ -46,11 +46,6 @@
 	 */
 	private ByteBuffer currentPage;
 
-	/**
-	 * Previously used pages that are available for reuse.
-	 */
-	private final Deque<ByteBuffer> reusablePages = new LinkedList<>();
-
```

Comment:
This patch is making significant changes to a commit in the same PR, could you squash them together?

Also the commit message doesn't make much sense :) I guess you simply mean you decided you prefer to keep it simple?

The JVM will always reuse blocks as necessary in some smart way, but that's not the same as reusing the same `pages` as we would have done here. You're allocating objects and buffers, let's not talk around it :D 

+1 to keep it simple and I agree that allocating these should not be a problem but that's for testing to measure.

Reply:
Ok, I'll squash it. The point of keeping the commit separate was merely to discuss the matter.